package/yast2.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Thu Jan  5 11:58:05 UTC 2017 - mfilka@suse.com
+
+- bnc#956755
+  - Use full type detection when caching network configuration to
+    solve ambiguos configurations (like bridge over wlan).
```

Comment:
ambiguo**u**s

But this is describing implementation details and tells the user nothing about the bug this is fixing or its symptoms.